packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Suspense } from 'react';
+import { Portal } from '../Portal/Portal';
+import { css, cx } from 'emotion';
+
+const modalBackdrop = css`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1040;
+  background-color: #f7f8fa;
+  opacity: 0.8;
+  backdrop-filter: blur(4px);
+`;
+
+interface Props {
+  title: string;
+  tabs: Array<{
```

Comment:
I think tabs do not belong to Modal. We should use composition here, tabs -> `content: JSX.Element`